ramani-maplibre/src/main/java/org/ramani/compose/MapLibreCompose.kt

```diff
@@ -49,10 +50,11 @@ internal suspend inline fun disposingComposition(factory: () -> Composition) {
 
 internal suspend fun MapView.newComposition(
     parent: CompositionContext,
-    style: Style,
+    styleBuilder: Style.Builder,
     content: @Composable () -> Unit,
 ): Composition {
     val map = awaitMap()
+    val style = map.awaitStyle(styleBuilder)
```

Comment:
Not sure if that's necessary, but it feels better to wait for the style here, to avoid issues like loading annotations before the style is fully loaded.

Reply:
@JonasVautherin Yes, I think this was actually the necessary part. Otherwhise you will get an exception when creating the AnnotationManager class below, because it requires the style to be loaded.